### PR TITLE
Update iOS Docs - Analytics

### PR DESCRIPTION
**Pull request by mlabieniec**
- Update copy for SDK and Amplify docs
- Add APNS setup guide
- Fix links

### CODE REVIEW

File: docs/sdk/analytics/fragments/ios/getting-started.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ Amazon Pinpoint is a fully managed AWS service that you can use to engage with y
 
 The Amplify CLI helps setup and configure Pinpoint within your application and connect with the AWS Mobile SDK.
 
-<b>Prerequisite:</b> [Install and configure the Amplify CLI](..)<br>
-<b>Recommendation:</b> [Complete the Getting Started guide](./start)
+ - **Prerequisite:** [Install and configure the Amplify CLI](/cli/start/install)
```
Comment (swaminator) [inline docs/sdk/analytics/fragments/ios/getting-started.md]: Only installing the CLI is a Prerequisite. The getting started guides per category should be focused solely on that (similar to the Auth one [here](https://docs.amplify.aws/lib/auth/getting-started?platform=js). Could you change the section to the blockquote markdown format:

```
> Prerequisite: [Install and configure the Amplify CLI](/cli/start/install)
```

File: docs/sdk/analytics/fragments/ios/getting-started.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ Amazon Pinpoint is a fully managed AWS service that you can use to engage with y
...
+ - **Recommendation:** [Complete the Getting Started guide](/start?integration=ios)
```
Comment (swaminator) [inline docs/sdk/analytics/fragments/ios/getting-started.md]: We do not need to guide them to the getting started anymore. It's clearly visible in the top nav.

File: docs/sdk/analytics/fragments/ios/getting-started.md [hunk 3]
```diff
@@ -139,11 +141,11 @@ Build and run your app to see usage metrics in Amazon Pinpoint. When you run the
 1. To see visualizations of the analytics coming from your app, open your project in the Amazon Pinpoint console by running the following:
 
 ```bash
-$ amplify console analytics
+$ amplify analytics console
```
Comment (swaminator) [inline docs/sdk/analytics/fragments/ios/getting-started.md]: I'm pretty sure its `amplify console analytics` @kaustavghosh06

Comment (mlabieniec) [reply docs/sdk/analytics/fragments/ios/getting-started.md]: This just worked for me, and I think makes more phonetic sense which is why we originally aliased the commands

Comment (swaminator) [reply docs/sdk/analytics/fragments/ios/getting-started.md]: Don't disagree or agree.. we're just using `amplify console <category>` in a bunch of the doc pages already.

**Pull request closed without merging**